pkg/ccr/job.go

```diff
@@ -946,6 +946,24 @@ func (j *Job) handleCreateTable(binlog *festruct.TBinlog) error {
 		return err
 	}
 
+	/*
+		Creating table will only occur when sync db.
+		When create view, the db name of sql is source db name, we should use dest db name to create view
+	*/
+	createSql := createTable.Sql
+	viewRegex := regexp.MustCompile("(?i)^CREATE(\\s+)VIEW")
+	isCreateView := viewRegex.MatchString(createSql)
+	if isCreateView {
+		log.Debugf("create view, use dest db name to replace source db name")
+
+		// replace `internal`.`source_db_name`. to `internal`.`dest_db_name`.
+		originalName := "`internal`.`" + strings.TrimSpace(j.Src.Database) + "`."
+		dbNameRegex := regexp.MustCompile(originalName)
+		replaceName := "`internal`.`" + strings.TrimSpace(j.Dest.Database) + "`."
+		createTable.Sql = dbNameRegex.ReplaceAllString(createSql, replaceName)
+		log.Debugf("original create view sql is %s, after repalce, now sql is %s", createSql, createTable.Sql)
+	}
+
```

Comment:
Move SQL related operations into Spec.go:CreateTable, and add a parameter 'srcDatabase'.

```go
createTable.Sql = dbNameRegex.ReplaceAllString(createSql, replaceName)
```

`strings.ReplaceAll` is enough here.

Reply:
done